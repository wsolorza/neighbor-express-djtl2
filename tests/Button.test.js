import { shallow } from "enzyme";
import Button from "../components/Button.jsx";

let wrapper;

beforeEach(() => {
  wrapper = shallow(
    <Button href='/volunteer'>
      Volunteer now!!
    </Button>
  );
});

describe("Button component", () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});