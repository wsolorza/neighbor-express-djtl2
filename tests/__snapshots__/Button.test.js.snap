// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders correctly 1`] = `
<p>
  <Link
    as="/volunteer"
    href="/[pid]"
  >
    <a
      className="usa-button usa-button--small"
      href="/volunteer"
    >
      Volunteer now!!
    </a>
  </Link>
</p>
`;
